src/main/kotlin/frc/robot/subsystems/superstructure/Superstructure.kt

```diff
@@ -1,7 +1,83 @@
 package frc.robot.subsystems.superstructure
-
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup
 import org.ghrobotics.lib.commands.FalconSubsystem
+import org.ghrobotics.lib.commands.FalconCommand
+import org.ghrobotics.lib.commands.sequential
+import org.ghrobotics.lib.mathematics.units.Meter
+import org.ghrobotics.lib.mathematics.units.SIUnit
+import org.ghrobotics.lib.mathematics.units.derived.Radian
+import org.ghrobotics.lib.mathematics.units.derived.radians
+import org.ghrobotics.lib.mathematics.units.inches
+import org.ghrobotics.lib.wrappers.FalconDoubleSolenoid
+import org.ghrobotics.lib.wrappers.FalconSolenoid
+import org.ghrobotics.lib.wrappers.hid.*
+import frc.robot.subsystems.superstructure.Elevator
+
+object Superstructure : FalconSubsystem() {
+
+    fun goToPreset(elevatorHeightPreset : SIUnit <Meter> , proximalAnglePreset : SIUnit <Radian> , wristAnglePreset : SIUnit <Radian>) {
+
+        var elevater = Elevator.master.encoder.position
+        var proximal = Proximal.ProximalMaster.encoder.position
+        var wrist = Wrist.wristMotor.encoder.position
+
+        if(elevater > 40.inches && elevatorHeightPreset > 40.inches){
+            //if high and going higher
+            //dose not matter no risk of hitting things
+            Proximal.ProximalPreset(proximalAnglePreset)
+            ElevatorPresets.elevatorGoToPreset(elevatorHeightPreset)
+            Wrist.wristPreset(wristAnglePreset)
```

Comment:
So here you instantiate 3 commands, but don't tell them to do anything. Put them into a parallel or sequential command group and then schedule that?

Reply:
done